_faqs/faq-12.md

```diff
@@ -1,8 +1,114 @@
 ---
-question: I want to cite RIOT. Which reference should I use?
+question: How does RIOT compare to other operating systems for embedded or IoT devices?
 ---
 
-Please consider the following references for citation.
+##### Disclaimer
 
-  * Emmanuel Baccelli, Cenk G&uuml;ndo&#287;an, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias W&auml;hlisch, <a href="{{ "assets/pdfs/riot-ieeeiotjournal-2018.pdf" | relative_url }}">RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT</a>, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.
+Developing an OS for embedded systems involves many trade-offs.
+Hence, there is not a single right choice for every application.
+In this comparison we try to give a balanced comparison of the high-level architectural trade-offs other operating systems took and how they compare to RIOT.
+Additionally, we compare the government of the project and the licensing.
 
+##### RIOT
+
+RIOT runs on many different CPU architectures by many different vendors, from [a tiny 8-bit MCU with 2 KiB RAM and 32 KiB flash][riot-arduino-nano] all the way up to high-performance [ARM Cortex M][riot-same54-xpro] and [RISC-V][riot-esp32c3-devkit] MCUs.
+Because RIOT abstracts the hardware differences by providing the same API for the same functionality across this large range of MCUs, projects building upon RIOT are highly portable and therefore resilient to changes in the supply chain.
+
+RIOT is designed to be a general-purpose OS with batteries included.
+It provides a large collection of in-tree modules and out-of-tree packages that seamlessly integrate in your application.
+This allows developers to pick the features they need from a large collection (as long as they can fit on the MCU).
+
+RIOT aims to be easy to learn by using sane defaults, by providing good documentation, and via its helpful community that happily answers questions [in the forum][riot-forum] or in the [matrix chat][riot-matrix].
+However, RIOT does not sacrifice advanced features, productivity or the ability to tinker with settings for more simplicity.
+This makes RIOT suitable for students in teaching, tinkerers in the DIY community, and professionals alike.
+
+RIOT is developed by a diverse, open, and [self-governed][riot-community-process] community.
+Members in the community mostly belong to one (or more) of the following groups: academia, industry, and makers/hobbyists.
+
+RIOT is distributed under a copyleft license and strongly prefers free and open source components over proprietary code and binary blobs.
+Still, RIOT does contain proprietary drivers (some even using binary blobs) where no free alternative is available, e.g. for the ESP WiFi chipset.
+<!-- FIXME: Make FAQ entries linkable and link to "Why LGPL?" -->
```

Comment:
I tested https://www.riot-os.org/#faq2 and it does not work for me. There also is no anchor in the HTML.

I think it would make sense to add the link, but making the FAQ entries linkable is IMO out of scope. (It would also make sense to have more stable IDs, as reordering the entries would break a number based system.)

Reply:
> I tested https://www.riot-os.org/#faq2 and it does not work for me. There also is no anchor in the HTML.

Huh? There is for me (see screenshot below) and it is also present in the Jekyll template https://github.com/RIOT-OS/riot-os.org/blob/be5b3902be5a318d29a0ef17ca048201ffdce238/index.html#L471

Reply:
Ah, it only works, if the FAQ is expanded... Ok, then let's postpone this.

Reply:
Same. If I use `<a id="faq-license">...</a>` it also works when the FAQ entry is not expanded.